src/services/web-monitoring-db.js

```diff
@@ -287,6 +287,10 @@ export default class WebMonitoringDb {
             queryList.push(`${encodedKey}[]=${encodeURIComponent(item)}`);
           }
         }
+        else if (key === CAPTURE_TIME) {
+          const formattedCaptureQuery = formatCaptureQuery(value);
+          queryList.push(formattedCaptureQuery);
```

Comment:
I think it might be a good idea to separate out the formatting of the value from the formatting of the key here, like we do above for arrays.